bindb.go

```diff
@@ -67,7 +84,82 @@ func LoadDB(dbpath string, autofix func(string) string) (*DB, error) {
 		}
 	}
 	_ = f.Close()
-	return db, nil
+	return nil
+}
+
+func LoadMulti(db *DB, path string, autofix func(string) string) (err error) {
+	var f *os.File
+	f, err = os.Open(path)
+	if err != nil {
+		return err
+	}
+	scanner := bufio.NewScanner(f)
+	scanner.Split(bufio.ScanLines)
+	var line string
+	var fields []string
+	for scanner.Scan() {
+		line = scanner.Text()
+		if autofix != nil {
+			line = autofix(line)
+		}
+		fields = strings.Split(line, "\t")
+		_, err = strconv.ParseInt(fields[0], 10, 32)
+		if err != nil || len(fields) < 12 {
+			fmt.Printf("BINDB row is not valid: %s, fields number is %d\n", line, len(fields))
+			continue
+		}
+		if item, exist := db.Map[fields[0]]; exist {
+			if item.Brand != strings.ToUpper(fields[1]) {
```

Comment:
These logic are little error prone when we change logic around upper case etc. Suggest we define `buildRecord` function and do following:

```
newRecord := buildRecord(fields)
if item, exist := db.Map[fields[0]]; exist {
  if item.Brand != newRecord.Brand {
    item.Brand = ""
  }
  // .. etc etc
} else {
  db.Map[fields[0]] = newRecord
}
```

also use `buildRecord` in `LoadMain`